exporters/stats/stackdriver/src/main/java/io/opencensus/exporter/stats/stackdriver/StackdriverExporterWorker.java

```diff
@@ -273,4 +306,103 @@ static String getDisplayNamePrefix(@javax.annotation.Nullable String metricNameP
       return metricNamePrefix;
     }
   }
+
+  @VisibleForTesting
+  static List<Metric> summaryMetricToDoubleGaugeMetric(Metric summaryMetric) {
```

Comment:
Why is not in the Util as all the others?

Reply:
Done,